p2p/libp2p/connectionsHandler.go

```diff
@@ -16,6 +16,7 @@ import (
 )
 
 const timeBetweenPeerPrints = time.Second * 20
+const kadProtocol = "/kad/1.0.0"
```

Comment:
shouldn't this be taken from the config?

Reply:
it is taken from config the protocol ID, this is just a suffix that the libp2p library adds it automatically